src/modules/ods/dto/get-ods-customer-response.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class GetOdsCustomerResponse {
+  @ApiProperty({
+    description: 'The unique UKEF ID of the customer',
+    example: '00312345',
+  })
+  readonly urn: string;
```

Comment:
Should we add same decorators to a response, might be worth checking CCM PRs.

Reply:
Decorators from classValidator aren't used on responses (as we wouldn't be validating that) so leaving this